docs/datasets.md

```diff
@@ -45,8 +45,8 @@ You can find it in your account's list of datasets.
 
 ```python
 >>> listing_resp = datasets.list()
->>> new_id in [ds['id'] for ds in listing_resp.json()]
-True
+>>> [ds['id'] for ds in listing_resp.json()]
+[...]
```

Comment:
What's the motivation for this change @KaiBot3000 @perrygeo? Tests don't pass otherwise?

It concerns me that we can't assert that a newly added dataset isn't in the list of datasets.